### PR TITLE
additional fixes based on feedback from Dave Maass

### PR DESCRIPTION
**Pull request by b-meson**

### CODE REVIEW

File: tactics/gun-shot-detectors.html [hunk 1]
```diff
@@ -43,25 +43,23 @@
     <div class="page-img"><img src="../img/Zapper_4.gif" alt="" style=""></div>
     <div class="page-txt">
         <h1 class="home-head">Gun Shot Detection</h1>
-        
-     <p>Shotspotter is an audio surveillance system. It consists of sensors - microphones - distributed over a neighborhood in order to determine the location of a gunshot. This sensor data can be used to distinguish gunshots from similar sounds such as fireworks and cars backfiring.</p>
+
+     <p>Shotspotter is the largest vendor selling its audio surveillance system. It consists of sensors - microphones - distributed over a neighborhood in order to determine the location of a gunshot. Sensor data is then sent through a set of algorithms and analysts to "verify" a gunshot. Verified alerts are then pushed to police in the field. Shotspotter claims to be able to distinguish gunshots from similar sounds such as fireworks and cars backfiring.</p>
```
Comment (ymaltsman) [inline tactics/gun-shot-detectors.html]: do you mean "largest vendor of audio surveillance systems"?

File: tactics/license-plate-readers.html [hunk 2]
```diff
@@ -43,35 +43,34 @@
     <div class="page-img"><img src="../img/Car_4.gif" alt="" style=""></div>
     <div class="page-txt">
         <h1 class="home-head">License Plate Readers</h1>
-        
-    <p>Automatic License Plate Recognition, or ALPR, take images of vehicles in public spaces, analyze the text on their license plates, and record the timestamp and position of the car (based on GPS coordinates of the camera).</p>
+
+    <p>Automatic License Plate Recognition, or ALPR, take images of vehicles in public spaces, analyze the text on their license plates, and record the timestamp and position of the car's license (based on GPS coordinates of the camera.)</p>
```
Comment (ymaltsman) [inline tactics/license-plate-readers.html]: "timestamp and position of their licenses" since we said "vehicles" and "license plates" in plural.

**Review by ymaltsman — COMMENTED** [review]
@b-meson looks good. Just added a couple small edits.

File: tactics/gun-shot-detectors.html (hunk 1 above)
Comment (b-meson) [reply tactics/gun-shot-detectors.html]: yep, I can fix

**Pull request merged**